gtk4/sys/src/lib.rs

```diff
@@ -1191,13 +1196,49 @@ pub struct _GtkATContextClass {
 
 pub type GtkATContextClass = *mut _GtkATContextClass;
 
+#[derive(Copy, Clone)]
 #[repr(C)]
-pub struct _GtkAccessibleInterface {
-    _data: [u8; 0],
-    _marker: core::marker::PhantomData<(*mut u8, core::marker::PhantomPinned)>,
+pub struct GtkAccessibleInterface {
+    pub g_iface: gobject::GTypeInterface,
+    pub get_at_context: Option<unsafe extern "C" fn(*mut GtkAccessible) -> *mut GtkATContext>,
+    pub get_platform_state:
+        Option<unsafe extern "C" fn(*mut GtkAccessible, GtkAccessiblePlatformState) -> gboolean>,
+    pub get_accessible_parent:
+        Option<unsafe extern "C" fn(*mut GtkAccessible) -> *mut GtkAccessible>,
+    pub get_first_accessible_child:
+        Option<unsafe extern "C" fn(*mut GtkAccessible) -> *mut GtkAccessible>,
+    pub get_next_accessible_sibling:
+        Option<unsafe extern "C" fn(*mut GtkAccessible) -> *mut GtkAccessible>,
+    pub get_bounds: Option<
+        unsafe extern "C" fn(
+            *mut GtkAccessible,
+            *mut c_int,
+            *mut c_int,
+            *mut c_int,
+            *mut c_int,
+        ) -> gboolean,
+    >,
```

Comment:
Interesting, I will look into adding subclassing support for this before doing the next release.